### PR TITLE
Adds an admin button to turn everyone into plasmamen with suits and internals

### PR DESCRIPTION
**Pull request by GuppyLaxx**
Boris told me if I made this I could finally use my 3 antag tokens

### CODE REVIEW

**Comment by DaedWren** [comment]
Skeleboners all round

**Comment by GuppyLaxx** [comment]
Tried so, doesn't replace the clothing, so you have plasmamen with no suit or helmet

**Comment by Barhandar** [comment]
Even after you've qdel'd said clothing already?

Also, Jordie probably means something like "new /datum/outfit/plasmaman/P;  H.equip_to_slot_or_del(new P.uniform(H), SLOT_W_UNIFORM)` except functional, i.e. using datum as the source of the type.

**Comment by Borisvanmemes** [comment]
oh my fuck he really made it.

**Comment by Jordie0608** [comment]
Yeah sorry I wasn't clear, @Barhandar is correct; you can condense all this into
````
H.set_species(/datum/species/plasmaman)
qdel(H.w_uniform)
qdel(H.wear_mask)
qdel(H.head)
H.drop_all_held_items()
var/datum/outfit/plasmaman/O = new /datum/outfit/plasmaman
H.equipOutfit(O, visualsOnly)
H.internal = H.get_item_for_held_index(2)
H.update_internals_hud_icon(1)
````
Although do test it first, should work but I can't be sure of that.

**Comment by zxaber** [comment]
Rather than delete all the worn gear, can it just be moved to the tile the player is standing on?

**Comment by Doppeltongue** [comment]
This is horrible. Speedmerge it.

File: code/modules/admin/secrets.dm [hunk 1]
```diff
@@ -593,6 +594,27 @@
...
+				var/helmet = H.head
+				qdel(helmet)
+				H.drop_all_held_items()
+				H.equip_to_slot_or_del(new /obj/item/clothing/under/plasmaman(H), SLOT_W_UNIFORM)
+				H.equip_to_slot_or_del(new /obj/item/clothing/head/helmet/space/plasmaman(H), SLOT_HEAD)
+				H.equip_to_slot_or_del(new /obj/item/clothing/mask/breath(H), SLOT_WEAR_MASK)
+				var/obj/tank = new /obj/item/tank/internals/plasmaman/belt/full
+				H.put_in_hands(tank)
+				H.internal = tank
+				H.update_internals_hud_icon(1)
```
Comment (ShizCalev) [inline code/modules/admin/secrets.dm]: TRUE

**Review by ShizCalev — CHANGES_REQUESTED** [review]
what Jordie said.

**Comment by AnturK** [comment]
:hourglass_flowing_sand:

**Pull request closed without merging**